rules/windows/execution_command_shell_started_by_svchost.toml

```diff
@@ -1,39 +1,40 @@
 [metadata]
 creation_date = "2020/02/18"
 maturity = "production"
-updated_date = "2021/03/03"
+updated_date = "2021/04/14"
 
 [rule]
 author = ["Elastic"]
 description = "Identifies a suspicious parent child process relationship with cmd.exe descending from svchost.exe"
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License v2"
 name = "Svchost spawning Cmd"
 risk_score = 21
 rule_id = "fd7a6052-58fa-4397-93c3-4795249ccfa2"
 severity = "low"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Execution"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  process.parent.name:svchost.exe and process.name:cmd.exe
+process where event.type in ("start", "process_started") and
+  process.parent.name : "svchost.exe" and process.name : "cmd.exe" and 
+  not process.command_line : "?:\\Windows\\SYSTEM32\\cmd.exe /c \"\"?:\\Program Files\\Npcap\\CheckStatus.bat\"\""
```

Comment:
2 `""` around the batch file?

```suggestion
  not process.command_line : """?:\\Windows\\SYSTEM32\\cmd.exe /c ""?:\\Program Files\\Npcap\\CheckStatus.bat"""""
```

ugh...

Reply:
![image](https://user-images.githubusercontent.com/64742097/114826014-3cdb7480-9dc7-11eb-8b86-2dabcd9d5c42.png)

![image](https://user-images.githubusercontent.com/64742097/114826156-6eecd680-9dc7-11eb-9721-bd973165b8ef.png)

this is a copy of the cmdline:

`
"C:\\Windows\\SYSTEM32\\cmd.exe /c \"\"C:\\Program Files\\Npcap\\CheckStatus.bat\"\""
`

Reply:
this works, but `process.args` is probably a bit cleaner

Reply:
```suggestion
  not (process.pe.original_file_name == "Cmd.Exe" and process.args : "?:\\Program Files\\Npcap\\CheckStatus.bat??")
```